io.github.congard.qnvsm.yml

```diff
@@ -0,0 +1,22 @@
+app-id: io.github.congard.qnvsm
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: qnvsm
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
why do you need this?

Reply:
Since it is a Qt app I assume I need access to x11, wayland and OpenGL for Qt (like in this example: https://docs.flatpak.org/en/latest/qt.html)
I need `--talk-name=org.freedesktop.Flatpak` because my app will call `nvidia-smi` system command

Reply:
so what happen if nvidia-smi isn't installed?

Reply:
It should be installed as a part of proprietary nvidia drivers. But if it isn't installed, then the app will notify that it cannot be used until the proprietary drivers (or any package that contains `nvidia-smi`) are installed